datafusion/sql/src/planner.rs

```diff
@@ -424,7 +424,7 @@ pub fn object_name_to_qualifier(
         .join(" AND ")
 }
 
-fn normalize_ident(id: Ident, enable_normalization: bool) -> String {
+pub fn normalize_ident(id: Ident, enable_normalization: bool) -> String {
```

Comment:
What would you think about making `crate::utils::normalize_ident` pub instead? making something public that only does something if `true` is passed in seems somewhat strange to me